standard.md

```diff
@@ -0,0 +1,94 @@
+# The definition of the standard
+
+## Glossary
+
+There are three levels of definitions in this standard:
+
+- **Must**: all diagnostic have to meet this conditions to comply with the standard.
+- **Recommend**: diagnostics are encouraged to meet them but they are not mandatory. Nevertheless, if a list of compatible diagnostics is compiled, those who will meet them will be highlighted.
+- **Suggest**: they will never ve enforced, but the team consider them as nice to have. They mostly correspond to quality of life improvements.
+
+In the same way, when defining the content of the yaml files, three different levels of definitions are  used:
+
+- **Required**: they must allways be present with the defined meaning and allowed values.
+- **Reserved**: they can be omitted, but in the case that are present, they must have the defined meaning and allowed values.
+- **Custom**: in all files, extra options can be added as needed by the diagnostics or the tools. The only requirement is that they do not collide with any `required` or `reserved` option.
+
+
+## The command line interface
+
+The diagnostic must be implemented as a command line tool that accepts a path to a yaml file as the only parameter.
+
+It is recommended that the diagnostic executable can also be called with the `--help` option. The diagnostic should then print on the console the following help information:
+
+- A small description of what the diagnostic does (reccomended)
+- A list of the options defined by the diagnostic to configure it. It is recommended that this list comes with a small explanation of it's meaning and effects, the type of the expected values and any default values taken when they are not explivitly defined (recommended).
+- A list of the mandatory common options required for the diagnostic to work (recommended).
+- A link to the online documentation, if available (suggested)
+
+## The diagnostic configuration file
+
+This section will define the syntax fot he main configuration file. It must be a yaml file containing a dictionary that contains a dictionary with all the options needed by a diagnostic
+
+### Required options
+
+The following options must be present in the configuration file passed to the diagnostic.
+
+- **input_files**: `list`. List of absolute paths to the metadata files describing the data to be used by the diagnostic. The format of this files is defined later.
+- **tool**: `str`. Name of the tool used to prepare the data for the diagnostic.
+- **version**: `str` Version of the tool used to prepare the data for the diagnostic.
```

Comment:
We have something similar in ESMValTool, in that case is mostly to provide shapefiles and such.

Do you think we need this as required or it will be enough to left it as reserved?

Reply:
'Reserved' is fine

Reply:
Added in reserverd